apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/NewFormButton/NewFormButton.tsx

```diff
@@ -26,53 +28,44 @@ export function NewFormButton(): ReactElement {
   const { t } = useTranslation('apps-journeys-admin')
   const { journey } = useJourney()
   const {
-    state: { selectedStep },
-    dispatch
+    state: { selectedStep }
   } = useEditor()
+  const { addBlock } = useBlockCreateCommand()
 
   async function handleClick(): Promise<void> {
```

Comment:
can you change these functions to 
```suggestion
  function handleClick(): void {
```